[2024-10-07T17:08:58.259+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: pinecone_embedding_test_dag.embed_convertapi_pages manual__2024-10-07T17:08:52.101252+00:00 [queued]>
[2024-10-07T17:08:58.268+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: pinecone_embedding_test_dag.embed_convertapi_pages manual__2024-10-07T17:08:52.101252+00:00 [queued]>
[2024-10-07T17:08:58.268+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-10-07T17:08:58.269+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 3
[2024-10-07T17:08:58.269+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-10-07T17:08:58.280+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): embed_convertapi_pages> on 2024-10-07 17:08:52.101252+00:00
[2024-10-07T17:08:58.285+0000] {standard_task_runner.py:55} INFO - Started process 2151 to run task
[2024-10-07T17:08:58.287+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'pinecone_embedding_test_dag', 'embed_convertapi_pages', 'manual__2024-10-07T17:08:52.101252+00:00', '--job-id', '51', '--raw', '--subdir', 'DAGS_FOLDER/Pinecone_Test.py', '--cfg-path', '/tmp/tmpf65urm90']
[2024-10-07T17:08:58.289+0000] {standard_task_runner.py:83} INFO - Job 51: Subtask embed_convertapi_pages
[2024-10-07T17:08:58.387+0000] {task_command.py:376} INFO - Running <TaskInstance: pinecone_embedding_test_dag.embed_convertapi_pages manual__2024-10-07T17:08:52.101252+00:00 [running]> on host 65f7b1aa1c1b
[2024-10-07T17:08:58.491+0000] {taskinstance.py:1590} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=pinecone_embedding_test_dag
AIRFLOW_CTX_TASK_ID=embed_convertapi_pages
AIRFLOW_CTX_EXECUTION_DATE=2024-10-07T17:08:52.101252+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-10-07T17:08:52.101252+00:00
[2024-10-07T17:08:59.107+0000] {logging_mixin.py:137} INFO - Error creating Pinecone index: (409)
Reason: Conflict
HTTP response headers: HTTPHeaderDict({'content-type': 'text/plain; charset=utf-8', 'access-control-allow-origin': '*', 'vary': 'origin,access-control-request-method,access-control-request-headers', 'access-control-expose-headers': '*', 'x-pinecone-api-version': '2024-07', 'X-Cloud-Trace-Context': 'f3decc83010ed8b239a4b2a4bda569ca', 'Date': 'Mon, 07 Oct 2024 17:08:59 GMT', 'Server': 'Google Frontend', 'Content-Length': '85', 'Via': '1.1 google', 'Alt-Svc': 'h3=":443"; ma=2592000,h3-29=":443"; ma=2592000'})
HTTP response body: {"error":{"code":"ALREADY_EXISTS","message":"Resource  already exists"},"status":409}
[2024-10-07T17:08:59.217+0000] {python.py:177} INFO - Done. Returned value was: None
[2024-10-07T17:08:59.237+0000] {taskinstance.py:1401} INFO - Marking task as SUCCESS. dag_id=pinecone_embedding_test_dag, task_id=embed_convertapi_pages, execution_date=20241007T170852, start_date=20241007T170858, end_date=20241007T170859
[2024-10-07T17:08:59.262+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2024-10-07T17:08:59.288+0000] {taskinstance.py:2623} INFO - 0 downstream tasks scheduled from follow-on schedule check
